hw1/02-palindrome.js

```diff
@@ -1,3 +1,53 @@
-const elem = document.querySelector('input');
+/**
+ * Checks if a string is a positive number or not.
+ *
+ * Examples:
+ *
+ * '0'    : false \
+ * 'abc'  : false \
+ * '-1'   : false \
+ * '001'  : true \
+ * '1.23' : true \
+ * 'Infinity' : true
+ */
+const isPos = function isPositiveNumber(str) {
+  if (!Number.isNaN(str) && Number(str) > 0) {
+    return true;
+  }
+  return false;
+};
 
-elem.addEventListener('input', handleInput);
+/**
+ * Checks if a string is a palindrome or not.
+ */
+const isPal = function isPalindrome(str) {
+  const len = str.length;
+  const halfway = Math.floor(len / 2);
+  for (let i = 0; i < halfway; i += 1) {
+    if (!(str[i] === str[len - i - 1])) {
+      return false;
+    }
+  }
+  return true;
+};
+
+const inputBox = document.querySelector('input');
+const result = document.getElementById('result');
+
+/**
+ * Display correct message in response to any user input into `inputBox`.
+ */
+const handleEvent = function handleEvent() {
```

Comment:
Honestly hard to find anything I'd recommend changing or that the Airbnb guide would recommend I think you did a good job of addressing the guide. I checked single vs double quote use, Array use, let vs var vs const use, Equality operators, and logical flow and it looks good to me. Your function declarations are also good I believe. Good job I don't see much that is wrong comparing your code to the guide. In attempt to offer something on line 42 "result.style.setProperty('color', 'darkred');" also works as "result.style.color = "darkred";" after the ".style" you should have access to any elements attributes such as color, backgroundColor, TextContent... etc just keep in mind the properties are camel case in this approach. Both approaches to setting css are perfectly fine I just wanted to give something as feedback.